### PR TITLE
Fix .spec file

### PR DESCRIPTION
**Pull request by socketpair**
So, it fixes some RPM linter things + simplify build + it works on Fedora 34 at least.

### CODE REVIEW

File: res/miraclecast.spec [hunk 1]
```diff
@@ -1,60 +1,57 @@
...
+%global commit 264a222d242734da369ca287aa6cfc6ca4f1f7bf
 %global shortcommit %(c=%{commit}; echo ${c:0:7})
 
 Name:           miraclecast
 Version:        1.0
-Release:        1.git%{shortcommit}%{?dist}
-Summary:        Connect external monitors to your system via Wifi-Display
-License:        LGPL
+Release:        2.git% {shortcommit}% {?dist}
+Summary:        Connect external monitors to your system via Wi-Fi display (miracast)
```
Comment (socketpair) [inline res/miraclecast.spec:7]: in order DNF search to work.

File: res/miraclecast.spec [hunk 2]
```diff
@@ -1,60 +1,57 @@
...
-%autosetup -n %{name}-%{shortcommit}
-
+%autosetup -n %{name}-%{commit}
 
 %build
-mkdir build
-cd build
-../autogen.sh g --prefix=%{_prefix}
+autoreconf -fiv
+%configure
```
Comment (socketpair) [inline res/miraclecast.spec:39]: do I need --enable-rely-udev ? how should I choose if I need or not ?

File: res/miraclecast.spec (hunk 1 above)
Comment (albfan) [reply res/miraclecast.spec]: got it

File: res/miraclecast.spec (hunk 2 above)
Comment (albfan) [reply res/miraclecast.spec:50]: add udev rules is not for everyone. So I think no

File: res/miraclecast.spec [hunk 3]
```diff
@@ -1,60 +1,68 @@
-%global commit c3c868e523f450ad4f0d77f5484a3b61f08120b7
+%global commit 264a222d242734da369ca287aa6cfc6ca4f1f7bf
 %global shortcommit %(c=%{commit}; echo ${c:0:7})
 
 Name:           miraclecast
 Version:        1.0
-Release:        1.git%{shortcommit}%{?dist}
-Summary:        Connect external monitors to your system via Wifi-Display
-License:        LGPL
+Release:        2.git% {shortcommit}% {?dist}
```
Comment (socketpair) [inline res/miraclecast.spec:6]: Spaces are BUGS! please dont merge

**Comment by albfan** [comment]
remove draft when is ready

**Comment by socketpair** [comment]
Did not understand. Right now you can merge, if you think it's OK.

вт, 11 янв. 2022 г., 21:19 Alberto Fanjul ***@***.***>:

**Comment by albfan** [comment]
This file shouldn't be maintained here, I have no idea how spec files works. Hope it helps anyone, but really that file should be in any other package, maintained by any other.

related with https://github.com/albfan/miraclecast/pull/309

How did you find it and how are you using it

**Comment by socketpair** [comment]
@albfan The file definitely should not be maintained here.
Unfortunately, I did not manage to get your project work (yes, I build rpm successfully), that's why I do not want to be a maintainer for this package in Fedora (but I can in general). If you can help me to get it working - I will make your package available in Fedora out of the box.

**Comment by albfan** [comment]
I suggest to open a proper issue with your details, there's lot of know how in this issue tracker.

**Pull request merged**